clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -22243,6 +22244,43 @@ llvm::CallInst *CodeGenFunction::EmitFPBuiltinIndirectCall(
                                    FPAccuracyIntrinsicID);
 }
 
+RValue CodeGenFunction::EmitIntelSYCLPtrAnnotationBuiltin(const CallExpr *E) {
+  const Expr *PtrArg = E->getArg(0);
+  Value *PtrVal = EmitScalarExpr(PtrArg);
+  auto &Ctx = CGM.getContext();
+
+  // Create the pointer annotation.
+  Function *F = CGM.getIntrinsic(llvm::Intrinsic::ptr_annotation,
+                                 {PtrVal->getType(), CGM.ConstGlobalsPtrTy});
+
+  SmallString<256> AnnotStr;
+  llvm::raw_svector_ostream Out(AnnotStr);
+
+  SmallVector<std::pair<std::string, std::string>, 4> Properties;
+
+  for (unsigned i = 1, e = E->getNumArgs(); i <= e / 2; i++) {
```

Comment:
Let's align with LLVM code style.
```suggestion
  for (unsigned I = 1, E = E->getNumArgs(); i <= e / 2; i++) {
```